topic04-git-introduction/book-git-tutorial/02.Branching.md

```diff
@@ -0,0 +1,40 @@
+# Git Branching
+
+----
+## What is Branching?
+
+
+This document is to show how branching works in git and how to use it. Branching is a feature available in git that lets you pull off from the master branch to make changes that won't directly effect the master branch unless merged. It also means that others have to review your work first just in case you have errors in your work such as spelling or bugs.
+
+## Why Would You Use Branching
+
+Branching is better than VCS in alot of different ways and one of them is disk space. Branching is all online and you don't have to keep track of your different versions of software which in turn will save space on your hard drive/ssd. 
+
+
+## Creating a Branch.
+Creating a branch is necessary if we want to make a push to the repo that you forked and are working on. Before we create a new branch we need to make sure we’re in the master branch To switch between branches we use git checkout <your-branch-name>. To create a branch we use the command git checkout -b <name your branch> this should now create a new branch in the forked repo.
```

Comment:
In the other prs commands are surrounded by backticks, lets follow the same syntax here.